Makefile

```diff
@@ -175,3 +175,13 @@ docker-publish-kusd:
 
 docker-publish-faucet:
 	docker push kowalatech/faucet
+
+## E2E tests
+
+GODOG_BIN := $(shell command -v godog 2> /dev/null)
+e2e:
```

Comment:
do we need to install godog everytime if we just want to execute tests?

Reply:
The godog installation is triggered only if the binary is not found.